saw-remote-api/python/saw/__init__.py

```diff
@@ -52,6 +55,27 @@ class VerificationResult(metaclass=ABCMeta):
     @abstractmethod
     def is_success(self) -> bool: ...
 
+class ProofResult(metaclass=ABCMeta):
+    goal: proofscript.ProofScript
+    valid: bool
+    counterexample: Optional[Any]
+
+    def is_valid(self) -> bool:
+        """Returns @True@ in the case where the given proof goal is valid, or true for
```

Comment:
I don't think `@` carries the same meaning in Python docstrings that it does in Haddock. I'd replace each `@` with ``` `` ``` for better Sphinx rendering.

Reply:
Good catch. I'll fix that.